site/content/docs/howto/supervisor/configure-supervisor-with-github.md

```diff
@@ -0,0 +1,177 @@
+
+[## Configure the Supervisor
+
+Create a [GitHubIdentityProvider](https://github.com/vmware-tanzu/pinniped/blob/main/generated/{{< latestcodegenversion >}}/README.adoc#githubidentityprovider) in the same namespace as the Supervisor.
+
+The simplest example uses https://github.com as the source of identity, and will allow any user to log in.
+Note that you do not need to explicitly specify a GitHub host since `github.com` is the default.
+
+```yaml
+apiVersion: idp.supervisor.pinniped.dev/v1alpha1
+kind: GitHubIdentityProvider
+metadata:
+  namespace: pinniped-supervisor
+  name: github-dot-com
+spec:
+  client:
+    secretName: github-dot-com-client-credentials
+  allowAuthentication:
+    organizations:
+      policy: AllGitHubUsers
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: pinniped-supervisor
+  name: github-dot-com-client-credentials
+type: secrets.pinniped.dev/github-client
+stringData:
+
+  # The "Client ID" from the GitHub App or GitHub OAuth App.
+  clientID: "<your-client-id>"
+
+  # The "Client secret" from the GitHub App or GitHub OAuth App.
+  clientSecret: "<your-client-secret>"
+```
+
+For another example, let's fill out all fields.
+Here we will configure a host and certificate for a GitHub Enterprise Server installation.
+
+```yaml
+apiVersion: idp.supervisor.pinniped.dev/v1alpha1
+kind: GitHubIdentityProvider
+metadata:
+  namespace: pinniped-supervisor
+  name: github-enterprise
+spec:
+  githubAPI:
+    
+    # Only the hostname or IP address and optional port, without the protocol.
+    # Pinniped will always use HTTPS.
+    host: github.enterprise.tld
+    
+    tls:
+      
+      # Must be a base64-encoded PEM cert ("-----BEGIN CERTIFICATE-----") or bundle
+      certificateAuthorityData: LS0tLS1CRUdJTiBDRVJUSUZJQ0FU....
+
+  client:
+    secretName: github-enterprise-client-credentials
+  allowAuthentication:
+
+    # "policy: OnlyUsersFromAllowedOrganizations" restricts authentication to only those users who belong in at least one of the "allowed" organizations.
+    # Additionally, their groups as presented to K8s will only reflect team membership within these organizations.
+    organizations:      
+      policy: OnlyUsersFromAllowedOrganizations
+      allowed:
+      - my-enterprise-organization
+      - admin-organization
+
+  claims:
+
+    # Use the login attribute of a user as the username to present to K8s.
+    # This attribute is taken from GitHub API endpoint [Get the authenticated user](https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#get-the-authenticated-user).
```

Comment:
We can drop the link since it will not render in a code block:
```bash
# This attribute is taken from GitHub API endpoint:
# https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#get-the-authenticated-user
```

Reply:
fixed